cpmpy/solvers/solver_interface.py

```diff
@@ -199,9 +199,17 @@ def solveAll(self, display=None, time_limit=None, solution_limit=None, **kwargs)
 
             Returns: number of solutions found
         """
-        # XXX: check that no objective function??
+        # TODO raise warning solver obj will be invalid? Or add assumption variables?
```

Comment:
Yes, Wout introduced a warnings.warn recently, we can do that here too, that would be good.